hash/sega_beena_cart.xml

```diff
@@ -60,23 +206,48 @@ license:CC0
 		<description>Soreike! Anpanman Hajimete Kaketa yo! Oboeta yo! Hiragana Katakana - Gojuuon Board Kinou-tsuki</description>
 		<year>2005</year>
 		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="それいけ！アンパンマン はじめてかけたよ!おぼえたよ!ひらがな・カタカナ～50音ボード機能つき～"/>
```

Comment:
You’ve got halfwidth exclamation marks here, too.  Also spurious spaces.

Reply:
Replaced all halfwidth with regular chars and removed spaces!